pfio/v2/local.py

```diff
@@ -4,6 +4,22 @@
 import shutil
 from typing import Optional
 
+try:
```

Comment:
This implementation should be in pfio/v2/_profiler.py or pfio/_profiler.py because these noop implementation will be used in all FS.